Microsoft.Toolkit.Uwp.UI.Controls.Layout/ContentSizer/ContentSizer.cs

```diff
@@ -30,7 +27,7 @@ public ContentSizer()
             // TODO: Can this be set in XAML, do we open a WinUI issue to track?
             ManipulationMode = ManipulationModes.TranslateX | ManipulationModes.TranslateY;
```

Comment:
Yes, I believe we could just set this in the style as `<Setter Property="ManipulationMode" Value="TranslateX,TranslateY"/>` and it should work. Didn't know that when I originally implemented it that commas should work for flag enums.